lib/ansible/plugins/inventory/azure_rm.py

```diff
@@ -226,6 +237,9 @@ def verify_file(self, path):
     def parse(self, inventory, loader, path, cache=True):
         super(InventoryModule, self).parse(inventory, loader, path)
 
+        if self.get_option('use_legacy_script_group_name_sanitization'):
```

Comment:
This isn't available until after self._read_config_data which set_options with the contents of the YAML config.

Reply:
yep, sorry. I actually hit that in integrated testing, but messed up the branch push. Updated now.